src/rand.rs

```diff
@@ -432,3 +435,32 @@ mod fuchsia {
         fn zx_cprng_draw(buffer: *mut u8, length: usize);
     }
 }
+
+#[cfg(target_os = "vxworks")]
```

Comment:
The supporting for VxWorks have been added in crate getrandom. The code here is copied from https://github.com/rust-random/getrandom/blob/master/src/vxworks.rs.